docs/preview/index.md

```diff
@@ -88,10 +97,11 @@ drwxr-xr-x 3 youruser youruser 4096 Feb 19 13:45 assets/
 drwxr-xr-x 4 youruser youruser 4096 Feb 19 13:45 docker/
 -rw-r--r-- 1 youruser youruser 2932 Feb 19 13:45 docker-compose.full.yml
 -rw-r--r-- 1 youruser youruser  943 Feb 19 13:45 docker-compose.yml
+-rw-r--r-- 1 youruser youruser 2665 Feb 19 13:45 docker-services.yml
```

Comment:
how interesting! This is not the ordering I get with `ls -la`. Probably because I am on Ubuntu and using the `fish` shell ;) .

Reply:
It is not the one I am getting either. I tried just adding and keeping the order that was there... We can change it for later.